tests/python/plot_dashboard_stress/main.py

```diff
@@ -68,6 +97,37 @@ def main() -> None:
     plot_paths = [f"plot_{i}" for i in range(0, args.num_plots)]
     series_paths = [f"series_{i}" for i in range(0, args.num_series_per_plot)]
 
+    if args.blueprint:
+        from rerun.blueprint import (
+            Blueprint,
+            BlueprintPanel,
+            Grid,
+            SelectionPanel,
+            TimeRangeBoundary,
+            TimeSeriesView,
+            VisibleTimeRange,
+        )
```

Comment:
nit: all our example code goes with `import rerun.blueprint as rrb` and uses that then

Reply:
Surely that cannot be true... I copy pasted that from somewhere else :smile:

Reply:
This guy it was https://github.com/rerun-io/rerun/blob/main/examples/python/blueprint/blueprint.py

Eh maybe I'll change em both then.